modules/indexer/code/elasticsearch/elasticsearch.go

```diff
@@ -179,12 +179,38 @@ func (b *Indexer) Index(ctx context.Context, repo *repo_model.Repository, sha st
 		reqs = append(reqs, b.addDelete(filename, repo))
 	}
 
+	// Helper function to support Go v1.2 and below
```

Comment:
Gitea are using go v1.21 at the moment so I don't think we need the function. Just use Go standard library.